tests/fixtures/tumor-variant-freq-study-result.yaml

```diff
@@ -0,0 +1,117 @@
+id: cancer_hotspots:CIC-R215
+type: TumorVariantFrequencyStudyResult
+name: Overall Tumor Variant Frequencr for CIC R215
+sourceDataSet:
+  type: DataSet
+  name: Cancer Hotspots
+  version: "2"
+specifiedBy:
+  type: Method
+  reportedIn:
+    type: Document
+    doi: 10.1158/2159-8290.CD-17-0321
+    name: Accelerating Discovery of Functional Mutant Alleles in Cancer
+affectedSampleCount: 25
+totalSampleCount: 24592
+affectedFrequency: 0.00101659
+focusVariant:
+  type: CategoricalVariant
+  name: CIC R215
+sampleGroup:
+  name: Overall
+  type: StudyGroup
+subGroupFrequency:
+  - type: TumorVariantFrequencyStudyResult
+    focusVariant:
+      type: CategoricalVariant
+      name: CIC R215W
+    affectedSampleCount: 17
+    totalSampleCount: 24592
+    affectedFrequency: 0.00069128
+    subGroupFrequency:
+      - type: TumorVariantFrequencyStudyResult
+        focusVariant:
+          name: CIC R215W
+          type: CategoricalVariant
+        sampleGroup:
+          id: remaining
+          name: CNS/Brain Tumor SubGroup
+          type: StudyGroup
+          characteristics:
+            - conceptType: Disease
+              name: CNS/Brain Tumors
+        affectedSampleCount: 16
+        totalSampleCount: 17
```

Comment:
This might not be correct. I would expect totalSampleCount to be the total number of samples of CNS/Brain Tumors in the full cancerhotspots dataset. This might be something that needs to be extracted from table S1 of [this publication](https://aacrjournals.org/cancerdiscovery/article/8/2/174/6249/Accelerating-Discovery-of-Functional-Mutant).